deployment-repository/src/main/java/org/jboss/as/repository/ContentRepository.java

```diff
@@ -365,18 +366,23 @@ public void removeContent(ContentReference reference) {
                 Path file = getDeploymentContentFile(reference.getHash(), true);
                 try {
                     Files.deleteIfExists(file);
+                    assert !Files.exists(file);
```

Comment:
This seems weird. I don't think this would ever happen. Also `Files.notExists()` is better IMO :)